test/test_inversion_solution.py

```diff
@@ -49,30 +50,104 @@ def test_load_crustal_from_archive(self, crustal_solution_fixture):
     #     assert isinstance(sol, InversionSolution)
     #     assert sol.fault_regime == 'SUBDUCTION'
 
-    def test_ruptures_intersecting_crustal(self, crustal_solution_fixture):
+    def test_get_rupture_ids_intersecting_crustal(self, crustal_solution_fixture):
         sol = crustal_solution_fixture
 
         WLG = location_by_id('WLG')
         # polygon = circle_polygon(5e5, -38.662334, 178.017654)
-        polygon = circle_polygon(1e5, WLG['latitude'], WLG['longitude'])  # 00km circle around WLG
+        polygon = circle_polygon(1e5, WLG['latitude'], WLG['longitude'])  # 100km circle around WLG
 
-        ruptures = sol.get_ruptures_intersecting(polygon)
+        ruptures = sol.get_rupture_ids_intersecting(polygon)
         all_rupture_ids = list(sol.ruptures.index)
 
         assert len(sol.ruptures) > len(ruptures)
         assert set(all_rupture_ids).issuperset(set(ruptures))
 
-    def test_get_ruptures_for_parent_fault(self, crustal_solution_fixture):
+    def test_get_rupture_ids_for_parent_fault(self, crustal_solution_fixture):
         sol = crustal_solution_fixture
 
-        df0 = set(sol.get_ruptures_for_parent_fault("Alpine Kaniere to Springs Junction"))
-        df1 = set(sol.get_ruptures_for_parent_fault("Alpine Jacksons to Kaniere"))
+        df0 = set(sol.get_rupture_ids_for_parent_fault("Alpine Kaniere to Springs Junction"))
```

Comment:
@DrCuriosity not your code I know, but can you rename these variables please  `df` prefix is for dataframes by convention

Reply:
Yes, thank you. I felt a bit conflicted about that at the time :-) Will sort.